modules/ref_runtimes_metering_labels.adoc

```diff
@@ -0,0 +1,17 @@
+//Include this reference module in product release notes.
+//Be sure you declare runtimes-attributes.doc
+//Content is intended for CADS documentation teams.
+
+[id='runtimes_metering_labels-{context}']
+= {ProductName} metering labels for Red Hat OpenShift
+
+The following metering labels apply to {ProductName} pods on OpenShift:
+
+* `com.redhat.component-name: {component-name}`
+* `com.redhat.component-type: {component-type}`
+* `com.redhat.component-version: {component-version}`
+* `com.redhat.product-name: {product-name}`
+* `com.redhat.product-version: {product-version}`
+
+You can add these labels to {ProductName} pods and then use the OpenShift Metering Operator to check your Red Hat subscription.
```

Comment:
To enable metering, you must add these labels to {ProductName} pods. You can then use the {openshiftshort} Metering Operator to check if you are in compliance with your Red Hat subscription.

Reply:
I think it's more of a "can" than a "must" but I'll double check with PM.

I also don't like the terminology "in compliance" because it sounds too much like something they'd say at a competitor who shall not be named. "Check your Red Hat subscription" is friendlier imo.

Reply:
From PM in the metering epic: "we need tell users that they can add the labels to their pods if they want to get metering coverage"

So this isn't mandatory action on the user's part. In future Operators will handle metering automatically. I think this module is just an interim solution until eng work is complete.

Reply:
@oraNod I used "must" because for now if users want Metering, they must the labels.

Reply:
but adding the labels doesn't enable metering

Reply:
As discussed @oraNod the final verdict :D:
If you want to use metering, add these labels to {ProductName} pods. You can then use the {openshiftshort} Metering Operator to check your Red Hat subscription.